.eleventy.js

```diff
@@ -8,6 +8,7 @@ const svgContents = require('eleventy-plugin-svg-contents');
 const codeClipboard = require('eleventy-plugin-code-clipboard');
 const { getLatestCdnVersion } = require('./utils/cdn-info');
 const { DateTime } = require('luxon');
+require('dotenv').config();
```

Comment:
This will probably only be used locally, on terraform we can set the environment variable directly. Could we potentially get rid of it to have less packages on production too?